### PR TITLE
update for DailyRollingFileAppender and log rotation

### PR DESCRIPTION
**Pull request by au-moon-yim**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.kafka_broker/templates/kafka_server_log4j.properties.j2 [hunk 1]
```diff
@@ -13,93 +13,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Unspecified loggers and loggers with additivity=true output to server.log and stdout
-# Note that INFO only applies to unspecified loggers, the log level of the child logger is used otherwise
-
-log4j.rootLogger=INFO, stdout, kafkaAppender
+log4j.rootLogger=INFO, stdout
```
Comment (eazamaAU) [inline roles/confluent.kafka_broker/templates/kafka_server_log4j.properties.j2:16]: Can you revert this change? It's preferable to have the rootLogger go to kafkaAppender, so that we collect all of the logs. If we don't, then some logs will get lost in the stdout void

File: roles/confluent.kafka_broker/templates/kafka_server_log4j.properties.j2 [hunk 2]
```diff
@@ -13,93 +13,72 @@
...
+
 #log4j.logger.kafka.network.Processor=TRACE, requestAppender
 #log4j.logger.kafka.server.KafkaApis=TRACE, requestAppender
 #log4j.additivity.kafka.server.KafkaApis=false
-log4j.logger.kafka.network.RequestChannel$=WARN, requestAppender
-log4j.additivity.kafka.network.RequestChannel$=false
+log4j.logger.kafka.request.logger=WARN, requestAppender
+log4j.additivity.kafka.request.logger=false
 
-log4j.logger.kafka.controller=TRACE, controllerAppender
```
Comment (eazamaAU) [inline roles/confluent.kafka_broker/templates/kafka_server_log4j.properties.j2:102]: Can you revert this back to trace? We can do a review of what gets logged later, but if Confluent defaults to trace, I'd like to see what's in there. I'm assuming there's a reason.

**Review by eazamaAU — CHANGES_REQUESTED** [review]

**Comment by au-moon-yim** [comment]
closed, will be updated soon.
